src/CMakeLists.txt

```
@@ -827,7 +827,7 @@ if (BUILD_SUPPLEMENTS)
 	# install more
 	foreach (_dir ${GMT_SUPPL_DIRS})
 		# get variables from subdirectory
-		get_subdir_var (_suppl_name SUPPL_NAME ${_dir})
+		get_subdir_var (_suppl_name SHARED_LIB_NAME ${_dir})
```

Here we need to get the value of SUPPL_NAME from src subdirectories.
```suggestion
		get_subdir_var (_suppl_name SUPPL_NAME ${_dir})
```